### PR TITLE
[release/6.0] Use 1ES machines for official builds

### PR DESCRIPTION
**Pull request by dougbu**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -188,7 +190,11 @@ stages:
 
         - job: Linux
           pool:
-            vmImage: ubuntu-latest
+            - ${{ if or(ne(variables['System.TeamProject'], 'internal'), in(variables['Build.Reason'], 'Manual', 'PullRequest', 'Schedule')) }}:
```
Comment (dougbu) [inline azure-pipelines.yml:193]:
```suggestion
            ${{ if or(ne(variables['System.TeamProject'], 'internal'), in(variables['Build.Reason'], 'Manual', 'PullRequest', 'Schedule')) }}:
```

File: azure-pipelines.yml [hunk 2]
```diff
@@ -188,7 +190,11 @@ stages:
...
+              vmImage: ubuntu-latest
+            - ${{ if and(eq(variables['System.TeamProject'], 'internal'), notin(variables['Build.Reason'], 'Manual', 'PullRequest', 'Schedule')) }}:
```
Comment (dougbu) [inline azure-pipelines.yml:195]:
```suggestion
            ${{ if and(eq(variables['System.TeamProject'], 'internal'), notin(variables['Build.Reason'], 'Manual', 'PullRequest', 'Schedule')) }}:
```

**Review by pranavkm — APPROVED** [review]
As long as @wtgodbe is happy 👍🏽

File: azure-pipelines.yml [hunk 3]
```diff
@@ -114,7 +114,9 @@ stages:
           - Windows
           - Source_Build_Complete
           pool:
-            vmImage: windows-latest
+            name: NetCore1ESPool-Internal
+            demands: ImageOverride -equals Build.Server.Amd64.VS2019
```
Comment (TanayParikh) [inline azure-pipelines.yml:118]: Is/will there be a VS2022 image?

```suggestion
            demands: ImageOverride -equals Build.Server.Amd64.VS2022
```

**Review by TanayParikh — APPROVED** [review]

File: azure-pipelines.yml (hunk 3 above)
Comment (dougbu) [reply azure-pipelines.yml:118]: There is a 2022 image. Why worry about VS version for this build❔

Comment (dougbu) [reply azure-pipelines.yml:118]: Turns out VS isn't used on CI at all. Arcade uses `dotnet msbuild` when global.json lacks a `tools.vs` entry, as in this repo. 2019 is fine&hellip;

Comment (TanayParikh) [reply azure-pipelines.yml:118]: Concern was mostly related to potential EOL of VS2019 images at some point.

**Pull request merged**